packages/serverless-orchestration/package.json

```diff
@@ -41,7 +41,7 @@
     "/src/**/*.js"
   ],
   "scripts": {
-    "test": "yarn truffle test test test/*.js"
+    "test": "yarn truffle test test/*.js"
```

Comment:
`yarn test test test test test`

TEST IT YARN!!

Reply:
hahaha yeah, I saw this in trying to debug those tests and I was like 🙃